.github/workflows/ci.yml

```
@@ -140,3 +140,31 @@ jobs:
         run: poetry run molecule test --scenario-name vars-priority
         env:
           TAILSCALE_CI_KEY: "${{ secrets.TAILSCALE_CI_KEY }}"
+
+  molecule-args:
+    runs-on: ubuntu-latest
```

Will need this given the new testing setup in this repo.

```suggestion
    environment: E2E
    runs-on: ubuntu-latest
```